docs/code-search/features.mdx

```diff
@@ -2,11 +2,15 @@
 
 <p className="subtitle">Learn and understand more about Sourcegraph's Code Search features and core functionality.</p>
 
+<LinkCards>
```

Comment:
Originally, this was at the top of the "Search Query Syntax" page. That page is an important technical reference, and is already a bit intimidating to read, so I felt it was best not to interrupt flow with a marketing callout.